common/transport/amqp/package.json

```diff
@@ -36,7 +36,7 @@
     "alltest": "istanbul cover node_modules/mocha/bin/_mocha -- --reporter spec test/_*_test*.js",
     "ci": "npm -s run lint && npm -s run build && npm -s run alltest-min && npm -s run check-cover",
     "test": "npm -s run lint && npm -s run build && npm -s run alltest && npm -s run check-cover",
-    "check-cover": "istanbul check-coverage --statements 95 --branches 83 --functions 92 --lines 96"
+    "check-cover": "istanbul check-coverage --statements 95 --branches 82 --functions 92 --lines 96"
```

Comment:
>82 [](start = 71, length = 2)

I see small movements here and there so I don't say much because it's always one percent, but after 5 reviews, it may become 5%. There is no good reason to see these going down and if there is we need to know where that percent point is lost and why we shouldn't care.